utilities/rpp-unittests/HOST_NEW/BatchPD_host_pln3.cpp

```diff
@@ -3666,6 +3691,90 @@ int main(int argc, char **argv)
     {
         // test_case_name = "color_jitter";
         missingFuncFlag = 1;
+        break;
+    }
+    case 82:
+    {
+        test_case_name = "ricap";
+        unsigned int initial_permute_array[images];
```

Comment:
Rpp32u is same as unsigned int, so change to Rpp32u. And RPP aims to standardize snake case for only function names, and camel case for variable names. So rename to Rpp32u initialPermuteArray[images] instead.